salt/roster/flat.py

```diff
@@ -43,11 +42,36 @@ def targets(tgt, tgt_type='glob', **kwargs):
                            **kwargs)
     conditioned_raw = {}
     for minion in raw:
-        conditioned_raw[str(minion)] = raw[minion]
+        conditioned_raw[str(minion)] = _resolve_sdb(raw[minion])
```

Comment:
I did not know that function existed. It does the exact same thing as `_resolve_sdb`, so I'll use `salt.config.apply_sdb`. Thanks!